gemfiles/rails_7.1.gemfile

```diff
@@ -0,0 +1,10 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "capybara", "~> 3"
+gem "rails", "~> 7.1.1"
```

Comment:
Please run `bundle exec appraisal generate` again, so this file is generated with your changes to `Appraisals`. Then, commit the changes.